src/Compute/Compute/ChangeLog.md

```diff
@@ -23,7 +23,9 @@
 * Added update functionality in `Update-AzVmss` for parameters `SecurityType`, `EnableSecureBoot`, and `EnableVtpm` for the parameter set with the Put operation.
 * Upgraded Azure.Core to 1.35.0.
 * [Breaking change] Removed unversioned and outdated linux image aliases of `CentOS`, `RHEL`, `UbuntuLTS` and `Debian`.
-* Removed unversioned and outdated images from New-AzVmss `-ImageName` argument completers.
+* [Breaking Change] Added defaulting logic for VM and VMSS creation to set SecurityType to TrustedLaunch and SecureBootEnabled and VTpmEnalbed to true when those are not set by the user.
+* [Breaking Change] Added defaulting logic for Disk creation to default to TrustedLaunch when able. Allows the user to turn this off by setting the SecurityType to Standard.
+* [Breaking Change] Removed the GuestAttestation extension installation by default for VM and VMSS resources. Removed the associated `DisableIntegrityMonitoring` parameter from `New-AzVM` and `New-AzVmss` cmdlets.
```

Comment:
I don't think this is mentioned in breaking change warnings: https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/upcoming-breaking-changes.md#azcompute

Reply:
@VeryEarly , correct. This was something we received last minute. If it's not good to remove, please let me know. Apparently this feature request was removed from other clients due to perf concerns but we were not made aware of this until this past week.

Reply:
No, you're right, I will add this back in.